core/atomic/flag.go

```diff
@@ -27,8 +27,8 @@ func (flag *Flag) IsSet() bool {
 // Toggle toggles the flag
 func (flag *Flag) Toggle(set bool) {
```

Comment:
can we rename this to `Set` as well? And create a new function `Toggle()` that simply switches between true and false?

Reply:
done, refactored the whole struct

Reply:
Renamed Toggle to SetValue. Toggle() was dropped as it is impossible to implement it with the atomic package functions and using mutex is not feasible as it will make the tx cache consume more RAM